models/articles.js

```diff
@@ -17,3 +17,24 @@ exports.selectArticleById = (article_id) => {
       }
     });
 };
+
+exports.updateArticleById = (article_id, inc_votes) => {
+  return db
+    .query(
+      `UPDATE articles 
+      SET votes = votes + $1
+      WHERE article_id = $2
+      RETURNING *;`,
+      [inc_votes, article_id]
+    )
+    .then((response) => {
+      if (response.rows.length === 0) {
+        return Promise.reject({
+          status: 400,
+          msg: 'bad request - invalid sytnax used for inc_votes on body',
```

Comment:
the query not returning rows isn't necessarily caused by the the inc_votes being wrong

it could also be because the article doesn't exist!